src/Aquifer.Common/Tiptap/TiptapModel.cs

```diff
@@ -17,20 +16,28 @@ public class TiptapRoot<TRootContent>
     public List<TRootContent> Content { get; set; } = [];
 }
 
-public class TiptapRootContent<TRootContent>
+public class TiptapNode<TRootContent>
 {
     public virtual string Type { get; set; } = null!;
-    public virtual TiptapAttributes? Attrs { get; set; }
+
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
Are these needed since it's the default option for `JsonUtilities.DefaultDeserialize`?

Reply:
I also changed some other stuff, but when I tried the first time I was getting a bunch of nulls back. I'll see if my other changes fixed that and I can undo these.

Reply:
@jwinston-bn just tested and it does still include the `null` without specifying this attribute:

<img width="705" alt="image" src="https://github.com/user-attachments/assets/a1dac9bd-40bf-45cd-8dc6-00029bc33594">